plutus-core/index-envs/src/Data/DeBruijnEnv.hs

```diff
@@ -33,22 +37,23 @@ instance DeBruijnEnv (BRAL.RAList a) where
     {-# INLINABLE cons #-}
     cons = BRAL.Cons
     {-# INLINABLE index #-}
-    index = BRAL.safeIndex
+    index = BRAL.safeIndexOne
```

Comment:
change the comment on the class to indicate that these count from 1

Reply:
Does this make a significant difference versus doing the subtraction before indexing?

Reply:
Maybe leave this optimization for a subsequent PR when we can benchmark just that change.

Reply:
it makes a difference , yes

Reply:
I did benchmark it before and makes a difference.

Reply:
ok i will leave it for another PR